en/thunderbird_privacy_policy.md

```diff
@@ -99,3 +103,16 @@ __Amazon Web Services__: Thunderbird uses Amazon Web Services (AWS) to host its
 
 __Email address providers (Desktop Only Legacy)__: Prior to version 128, Thunderbird partnered with Gandi.net and Mailfence to allow you to create a new email address through Thunderbird. If you choose to use this feature, your email address search terms are sent to Gandi.net and Mailfence to return available addresses. In addition, your country location is also shared to provide the correct prices. You can learn more about [Gandi.net’s](https://contract.gandi.net/v5/contracts/14420/Privacy_Policy_US_2.0_en.pdf) and [Mailfence’s](https://mailfence.com/en/privacy.jsp) data practices by reading their privacy notices.
 
+## Contact Us
+
+If you want to make a correction to your information, or you have any questions about our privacy policies, please get in touch with:
+
+__MZLA Technologies Corporation__
+Attn: Mozilla - Privacy
+149 New Montgomery St, 4th Floor,
+San Francisco, CA 94105
+USA
+compliance@mozilla.com
+
+[See here for Data Subject Access Requests](https://privacyportal.onetrust.com/webform/1350748f-7139-405c-8188-22740b3b5587/4ba08202-2ede-4934-a89e-f0b0870f95f0). If you are under 13, we don’t want your personal information, and you must not provide it to us. If you are a parent and believe that your child who is under 13 has provided us with personal information, please contact us to have your child’s information removed.
+For product support requests, please [visit our forums](https://support.mozilla.org/).
```

Comment:
At risk of pointing out the obvious, this line break won't have much effect in markdown, it will just become an empty space after `removed.`.

Reply:
Good catch, I didn't notice that

Reply:
Do you want me to fix it or publish it as is? Just line break or new paragraph?

Reply:
If you could fix it and publish it it would be great.
Thank you so much!

Reply:
A new paragraph is good, thanks

Reply:
Looks like I don't permissions to push to your branch (it's also on main, so you might want to remove the fork and refork it). I'll fix after the merge.